pkg/apis/triggers/v1alpha1/event_listener_defaults.go

```diff
@@ -5,4 +5,23 @@ import (
 )
 
 // SetDefaults sets the defaults on the object.
-func (el *EventListener) SetDefaults(ctx context.Context) {}
+func (el *EventListener) SetDefaults(ctx context.Context) {
+	if IsUpgradeViaDefaulting(ctx) {
+		// Most likely those EventListener passed here is already running
```

Comment:
"most likely the EventListeners passed here are already running"?

Reply:
:face_with_head_bandage: